app/serializers/move_serializer.rb

```diff
@@ -50,15 +50,6 @@ class MoveSerializer < ActiveModel::Serializer
     allocation: %i[to_location from_location moves_count created_at],
   }.freeze
 
-  def person
```

Comment:
The move model defines a :through relationship so this isn't needed

See: 

https://github.com/ministryofjustice/hmpps-book-secure-move-api/blob/dbf4d75b6ccc0ca24ef14640f92394532c4e1653/app/serializers/move_serializer.rb#L21
https://github.com/ministryofjustice/hmpps-book-secure-move-api/blob/dbf4d75b6ccc0ca24ef14640f92394532c4e1653/app/models/move.rb#L50